libs/community/tests/integration_tests/document_loaders/test_sqlalchemy_sqlite.py

```diff
@@ -0,0 +1,190 @@
+"""
+Test SQLAlchemy/SQLite document loader functionality.
+"""
```

Comment:
It seems the tests are the same for sqlite and postgresql.
Maybe they could be factorized ?

Reply:
Working on it.

Reply:
The test cases have been refactored accordingly on behalf of 1f21b47a3809. Thanks for your suggestion.